packages/core/src/protocol-helpers.ts

```diff
@@ -139,9 +139,7 @@ export default class ProtocolHelpers {
         if (
             actionName != null &&
             uriScheme != null &&
-            td?.actions != null &&
-            td.actions[actionName] != null &&
-            td.actions[actionName].forms != null &&
+            td?.actions[actionName]?.forms != null &&
```

Comment:
Hmm, okay, sorry, this seems to have introduced an error

Reply:
Jupp, I am reverting one step